src/pages/Links/inviteComposer.module.scss

```
@@ -12,3 +12,47 @@
+
+    &:hover,
+    &:focus {
+        background: var(--m3-surface-container, #f0edee);
+    }
+
+    &:focus-visible {
+        outline: 2px solid var(--m3-primary, #a5000a);
+        outline-offset: -4px;
+    }
+
+    :global(.anticon) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)